testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -602,18 +602,18 @@ protected QualifiedObjectName getQualifiedTableName(String tableName)
                 tableName);
     }
 
-    private TableMetadata getTableMetadata(TableHandle tableHandle)
+    public static TableMetadata getTableMetadata(Session session, QueryRunner queryRunner, TableHandle tableHandle)
```

Comment:
`AbstractTestQueryFramework` doesn't really have any static methods (it's not utility class). I would just copy this method to `AbstractCostBasedPlanTest`